setup.py

```diff
@@ -67,7 +67,7 @@ def version():
         "machine-learning",
         "experiment-tracking",
     ],
-    python_requires=">=3.9, <3.12",
+    python_requires=">=3.9, <3.13",
```

Comment:
The PR title says "[SDK] Bump min python version ..." - should be **max**, right?

Reply:
right 👍